### DISCUSSION TITLE
Using Twin and VSCode Inline Fold to only fold Tailwind classes

### DISCUSSION THREAD

**Discussion by colemilne54** [ben-rogerson/twin.macro#809] [Show and tell]
Pairing twin with VSCode Inline Fold allows you to fold Tailwind classes while keeping your custom classes visible.

[Demo/ How-to](https://github.com/colemilne54/twin-fold)

**Comment by ben-rogerson** [comment]
Hey this is pretty cool - how hard would it be to add it to to styled.element syntax?

eg:

```js
styled.div(...)
styled.div`...`
```
